embedchain/cache.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os  # noqa: F401
+from typing import Any, Dict
+
+from gptcache import cache
+from gptcache.adapter.adapter import adapt  # noqa: F401
+from gptcache.config import Config  # noqa: F401
+from gptcache.manager import get_data_manager
+from gptcache.manager.scalar_data.base import Answer
+from gptcache.manager.scalar_data.base import DataType as CacheDataType
+from gptcache.session import Session
+from gptcache.similarity_evaluation.distance import \
+    SearchDistanceEvaluation  # noqa: F401
+
+
+def gptcache_pre_function(data: Dict[str, Any], **params: Dict[str, Any]):  # noqa: F401
+    return data["input_query"]
+
+
+def gptcache_data_manager(vector_dimension):  # noqa: F401
```

Comment:
Do you need `  # noqa: F401`? Ideally should not be here.

Reply:
Not needed will clean up.